dtfs-central-api/src/helpers/get-specific-tfm-facility-values.ts

```diff
@@ -4,34 +4,32 @@ import { TfmFacilitiesRepo } from '../repositories/tfm-facilities-repo';
 import { NotFoundError } from '../errors';
 import { convertTimestampToDate } from './convert-timestamp-to-date';
 import { getEffectiveCoverEndDateAmendment } from './amendments/get-effective-cover-end-date-amendment';
+import { SpecificTfmFacilityValues } from '../types/tfm/tfm-facility';
 
 /**
  * Gets the latest values for the TFM facility with the supplied facility id
  * @param facilityId - The facility id
  * @param reportPeriod - The report period
  * @returns The latest values
  */
-export const getLatestTfmFacilityValues = async (
-  facilityId: string,
-  reportPeriod: ReportPeriod,
-): Promise<{
-  coverEndDate: Date;
-  coverStartDate: Date;
-  dayCountBasis: number;
-  interestPercentage: number;
-  coverPercentage: number;
-}> => {
+export const getSpecificTfmFacilityValues = async (facilityId: string, reportPeriod?: ReportPeriod): Promise<SpecificTfmFacilityValues> => {
   const tfmFacility = await TfmFacilitiesRepo.findOneByUkefFacilityId(facilityId);
+
   if (!tfmFacility) {
-    throw new NotFoundError(`Failed to find a tfm facility with ukef facility id '${facilityId}'`);
+    throw new NotFoundError(`TFM facility ${facilityId} could not be found`);
```

Comment:
tfm facility's have a facility id and a ukef facility id and they are different so in this context we should be more explicit: 
```suggestion
    throw new NotFoundError(`TFM facility with ukef facility id '${facilityId}' could not be found`);
```